packages/swingset-liveslots/src/vatstore-usage.md

```diff
@@ -112,7 +114,7 @@ For each virtual object kind that is defined, we store a metadata record for pur
 
 # Virtual/Durable Collections (aka Stores)
 
-Liveslots provides a handful of "virtual collection" types to vats, to store high-cardinality data on disk rather than in RAM. These are also known as a `Store`. They provide limited range queries and offer a single fixed sort index: numbers sort as usual, BigInts sort as usual but separate from numbers, strings sort lexicographically by UTF-8 encoding, and object references sort by insertion order.
+Liveslots provides to vats a handful of "virtual collection" types for storing high-cardinality data on disk rather than in RAM. Each instance of one of these types is described as a "[Store](../../store/docs/store-taxonomy.md)". They provide limited range queries and offer a single fixed sort index: numbers sort as usual, BigInts sort as usual but separate from numbers, strings sort lexicographically by UTF-8 encoding, and object references sort by insertion order.
```

Comment:
nit, maybe "Liveslots provides vats with a handful..", the indirect/passive voice sounds weird to me